pydantic/__init__.py

```diff
@@ -1,3 +1,5 @@
+import importlib
```

Comment:
is there a particular reason this needs to be imported here?

Reply:
For `patch_importlib_with_migration_info(importlib)` it needs the importlib to be in the root of the package unfortunately